dags/analysis.py

```diff
@@ -29,7 +29,8 @@
         image=anomdtct_image,
         email=["ascholtz@mozilla.com", "jmccrosky@mozilla.com",],
         arguments=["{{ds}}"]
-        + ["--spreadsheet-id=" + Variable.get('anomdtct_spreadsheet_id')],
+        + ["--spreadsheet-id=" + Variable.get('anomdtct_spreadsheet_id')]
+        + ["--spreadsheet-key=" + Variable.get('anomdtct_spreadsheet_api_key')],
```

Comment:
Variables the contain some key words such as `api_key` get automatically masked. So they appear as *****